doc/safety/safety_overview.rst

```diff
@@ -85,6 +85,22 @@ implementation of safety-related systems that aims to reduce the risk of acciden
 overall safety. By following the standard, organizations can ensure that their safety-related
 systems are designed and implemented to the highest level of safety integrity.
 
+Why IEC 61508?
+==============
+The IEC 61508 standard was chosen because it is a basic functional safety standard that is
+applicable to all kinds of industry sectors and which can server as basis for that various industry
+standards. Therefore, it is a good standard for zephyr as well, because zephyr can be used in many
+different industry sectors.
+
+The following picture shows the relation between the IEC 61508 standard and other standards:
```

Comment:
On my phone so this is missing 100-char line wraps 
 
```suggestion
The IEC 61508 standard was selected because it serves as a foundational functional safety standard applicable across various industry sectors. It provides a robust framework that can be adapted to create specific standards for different industries. This makes IEC 61508 particularly relevant for Zephyr, as the operating system's versatility allows it to be effectively utilized across a wide range of industry sectors.

The following diagram illustrates the relationship between the IEC 61508 standard and other related standards:
```

Reply:
Strange on my side it is 100-char line wrapped

Reply:
No I meant the "suggestion" I was making had no line wraps so basically hinted at making sure you would apply line wrap on your side if you were to accept the suggestions :)

Reply:
Oh okay misunderstood it and only saw the line wrap in the suggestion I will apply the suggestion and the line wraps sorry :)